README.md

```diff
@@ -48,6 +48,18 @@ You'll need the webhook URL to instantiate the client (or for the configuration
 
 ### Instantiate the client
 
+Using Slack's [OAuth token](https://api.slack.com/messaging/sending) method:
```

Comment:
The link does not seem to show oauth usage directly.

Reply:
I've updated this.